File: Python_code/testForFile.py
import suds
import base64
from suds.client import Client
fileContent = open('d:\\2.pdf', 'rb').read()
base64Str = base64.b64encode(fileContent)
username='u1xxxxx'
pwd = 'xxxxxx'

url = 'http://service.ygys.net/resumeservice.asmx?WSDL'
client = Client(url)
result = client.service.TransResumeByJsonStringForFileBase64(username,pwd,base64Str,'.pdf')

print result

File: Nodejs_code/testForFile.js
var fs = require('fs'); 
var soap = require('soap');
var url = 'http://service.ygys.net/resumeservice.asmx?WSDL';

var data = fs.readFileSync('d:\\xxx.pdf');  // 同步读取本地文件
var base64str = new Buffer(data).toString('base64');  // 转成base64字符串
var args = { username: 'u100xxx', pwd: 'xxx', content: base64str, ext: '.pdf' };  // 构建参数

soap.createClient(url,function(err,client) {
  if (err) {
    console.log(err);
  }

  client.ResumeService.ResumeServiceSoap12.TransResumeForFileBase64(args, function(err, result){
    console.log(result);
  });

});
